ts/src/alpaca.ts

```diff
@@ -567,7 +910,7 @@ export default class alpaca extends Exchange {
         return this.parseOrderBook (rawOrderbook, market['symbol'], timestamp, 'b', 'a', 'p', 's');
     }
 
-    async fetchOHLCV (symbol: string, timeframe = '1m', since: Int = undefined, limit: Int = undefined, params = {}): Promise<OHLCV[]> {
+    async fetchOHLCV (symbol: string, timeframe = '1m', since: Int = 0, limit: Int = 1000, params = {}): Promise<OHLCV[]> {
```

Comment:
we shouldn't change the default value for the parameter, if needed you can do inside the function

```
if (limit === undefined ) { limit = 1000}
````

Reply:
Cool. I was unsure about that doing that, but I found some that had been changed from the defaults, so I thought it was ok. Will revert back.